CODE_OF_CONDUCT_FR.md

```diff
@@ -4,4 +4,68 @@
 
 ***Note :** ce document est une traduction du [GSRI Code of Conduct](./CODE_OF_CONDUCT.md). En cas d'incohérence entre les deux documents, c'est la version anglophone qui s'applique. Cette version est fournie pour faciliter la compréhension mais n'a pas la valeur probante du document anglophone.*
 
-👷 ***Nous sommes actuellement en train de traduire ce document***
\ No newline at end of file
+## Qui sommes-nous ?
+
+Le `Groupe Special de Reconnaissance et d'Intervention` est une équipe ArmA 3 française, orientée milsim. Nous sommes une organisation sans but lucratif composée de membres bénévoles, qui contribuent au développement d'outils et de mods pour le jeu Arma 3.
+
+Nous jouons des missions, privées comme publiques, orientées forces spéciales, à un haut niveau de discipline et de réalisme. Davantage d'information est disponible sur notre [site internet officiel](https://www.gsri.team) et sur notre [serveur Discord](https://discord.gg/bhMn4jd). L'un comme l'autre sont consultables uniquement en français.
+
+## Notre engagement
+
+Dans le but de favoriser un environnement ouvert et accueillant, nous, contributeurs, nous engageons à faire de notre participation dans notre projet et notre communauté une expérience libre de tout harcelement pour tout invididu.
+
+## Note importante sur le contenu violent
+
+Cette communauté est rassemblée autour d'un jeu vidéo **catégorisé M par l'ESRB**, du fait de la présentation *de violence intense, de sang, de langage grossier*. C'est pourquoi notre communauté fait l'objet d'une modération appropriée pour des personnes agées de 17 ans et plus. Cela ne singifie pas qu'elle est inappropriée pour des personnes plus jeunes, mais du contenu susceptible de heurter leur sensibilité risque d'être partagé. Par ailleurs, le contenu inapproprié ne sera pas modéré aussi rapidement et efficacement que ce que l'on serait en droit d'attendre d'une équipe de modération travaillant pour une communauté entièrement publique.
```

Comment:
```suggestion
Cette communauté est rassemblée autour d'un jeu vidéo **catégorisé M par l'ESRB**, du fait de la présentation *de violence intense, de sang, de langage grossier*. C'est pourquoi notre communauté fait l'objet d'une modération appropriée pour des personnes agées de 17 ans et plus. Cela ne singifie pas qu'elle est inappropriée pour des personnes plus jeunes, mais du contenu susceptible de heurter leur sensibilité risque d'être partagé. Par ailleurs, le contenu inapproprié ne sera pas modéré aussi rapidement et efficacement que ce que l'on serait en droit d'attendre d'une équipe de modération travaillant pour une communauté tout public.
```

Reply:
If I'm right, there is no change in this suggestion ?